Source/VisualRecognitionV3/VisualRecognition+CoreML.swift

```diff
@@ -217,10 +217,6 @@ extension VisualRecognition {
 
         // return results after all classification requests have executed
         dispatchGroup.notify(queue: DispatchQueue.global(qos: .userInitiated)) {
-            guard errors.isEmpty else {
```

Comment:
Should we check for `results` being non empty here before trying to convert?

Reply:
Yes we should. Thanks for catching that. I'll push a change up shortly to address that.